src/gui/guiutility.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) by Christian Kamm <mail@ckamm.de>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#ifndef GUIUTILITY_H
+#define GUIUTILITY_H
+
+#include <QString>
+
+namespace OCC
+{
+namespace Utility
+{
+
+/** Copies a string to the clipboard.
+ *
+ * Includes a workaround for
+ * https://github.com/owncloud/client/issues/3300
```

Comment:
The bug https://bugreports.qt.io/browse/QTBUG-48953 has been  fixed in Qt 5.6.1,  so do we still need it as we only use Qt 5.6 on mac anyway?

In that case, this function would not be needed, as this is just a wrapper for QApplication::clipboard()->setText(...);